src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/persistence/PublisherEventsPersistor.java

```diff
@@ -41,33 +41,34 @@ public PublisherEventsPersistor(final Persistable persistable) {
         this.persistable = persistable;
     }
 
-    public void persistAction(final Action action) {
-        long timestamp = Instant.now().toEpochMilli();
-        LOG.debug("Action: [{}] published to persistor publisher.", action.name());
-        PerformanceAnalyzerApp.RCA_RUNTIME_METRICS_AGGREGATOR.updateStat(
-                RcaRuntimeMetrics.ACTIONS_PUBLISHED, action.name(), 1);
-        if (action.impactedNodes() != null) {
-            final String nodeIds = action.impactedNodes().stream()
-                                           .map(n -> n.getNodeId().toString())
-                                           .collect(Collectors.joining(",", "{", "}"));
-            final String nodeIps = action.impactedNodes().stream()
-                                           .map(n -> n.getHostAddress().toString())
-                                           .collect(Collectors.joining(",", "{", "}"));
```

Comment:
I don't have a strong opinion either way - i think the old format was also ok.
But this format is expected by multiple integ tests. Are they passing with your changes? Please confirm and update any breaking ITs.

Reply:
While I agree the new format might be simpler. But reverting to old format to reduce the effort of modifying ITs and push it to prod for release purposes. Updated the local build pass message in the PR description.